src/pseudopeople/constants/data_values.py

```diff
@@ -0,0 +1,42 @@
+from typing import Dict
+
+import pandas as pd
+
+from pseudopeople.constants.metadata import DatasetNames
+
+# Targeted omission constants for do_not_respond
+DO_NOT_RESPOND_BASE_PROBABILITY = 0.0024
+
+DO_NOT_RESPOND_ADDITIVE_PROBABILITY_BY_RACE: Dict[str, float] = {
+    "AIAN": 0.0067,
+    "Asian": -0.0286,
+    "Black": 0.0306,
+    "Latino": 0.0475,
+    "Multiracial or Other": 0.041,
+    "NHOPI": -0.0152,
+    "White": -0.0188,
+}
+
+DO_NOT_RESPOND_ADDITIVE_PROBABILITY_BY_SEX_AGE: pd.DataFrame = pd.DataFrame(
```

Comment:
I think it might be simpler for you to have this be a dictionary rather than a `DataFrame`. Like this:

```
DNRAPBSA: Dict[str, pd.Series] = {
    "Female": pd.Series([0.0255, -0.0014, ...], index=[pd.Interval(0, 4), pd.Interval(5, 9), ...]),
    "Male": pd.Series([0.0255, -0.0014, ...], index=[pd.Interval(0, 4), pd.Interval(5, 9), ...]),
}
```

I believe this will simplify a lot of the logic in your noise function and you'll have to do a lot less pandas manipulations, since what you really want is a sex-specific series with the age interval in the index, which is how it's defined in my suggestion.

Reply:
If you think that's simpler I can make that change. I think that simplicity is maybe in the eye of the beholder.